eclass/meson.eclass

```diff
@@ -379,13 +369,32 @@ meson_src_configure() {
 	python_export_utf8_locale
```

Comment:
This one's redundant to what python eclasses do.

Reply:
I didn't specifically change this but I'd be happy to remove it as I wasn't sure myself why it should be needed.